gemrb/GUIScripts/GUICommon.py

```diff
@@ -408,12 +408,15 @@ def GetActorClassTitle (actor):
 				# first (previous) kit or class of the dual class
 				if Dual[0] == 1:
 					ClassTitle = CommonTables.KitList.GetValue (Dual[1], 2)
-				elif Dual[0] == 2:
+				else:
 					ClassTitle = CommonTables.Classes.GetValue (GetClassRowName(Dual[1], "index"), "CAP_REF")
 				if ClassTitle != "*":
 					ClassTitle = GemRB.GetString (ClassTitle)
 				ClassTitle += " / "
-				ClassTitle += CommonTables.Classes.GetValue (GetClassRowName(Dual[2], "index"), "CAP_REF", GTV_REF)
+				if Dual[0] == 3:
+					ClassTitle += CommonTables.KitList.GetValue (Dual[2], 2, GTV_REF)
```

Comment:
Please use named row/column lookups wherever you can. We specifically moved to that to be safe about any table changes and crossgame differences.

Reply:
Oh, sorry, was not aware.  Does this also apply for C++ code?  I might need to redo a couple of functions if so.

Reply:
It does, but there we try to preload everything. That one call in core getkitindex or whatwasit is an outlier in both senses.

Reply:
OK, then I'll add lookups in Python and preloads in core.

Reply:
Wait a bit with this dualclass stuff. I feel like it can be redesigned, but need to both read the rest of it thoroughly and think about other options. The aforementioned MC bit would only take care of a part.